test/suites/ACID.Octave.3.8.0.md5

```diff
@@ -58,7 +58,7 @@ scatterPlotRandom : 6fadda0940c7c468a1aee0aef8203c4e
 sine_with_annotation : 966fbd52d16fb908384adb9ad3f8d98b
 sine_with_markers : 33e7e752de0d54a80fd95a70fa17a1ef
 spherePlot : 4f8881f1d81962a213aa0526566f976b
-stackedBarsWithOther : 766e986fb790d41c575cb10308025a37
+stackedBarsWithOther : 9fd47e847290b1d5ec435c1279e6e978
```

Comment:
Do you have an idea why the Octave hash has changed, while the code change is MATLAB-only (i.e. in `findLegendInfoMATLAB`)? Or is this a pre-existing mismatch?